### DISCUSSION TITLE
Material de Apoyo

### DISCUSSION THREAD

**Discussion by JoseAngelSilva2022** [LaunchX-InnovaccionVirtual/CursoIntroPython#216] [Ideas]
Hola a todos de la comunidad, espero que todos ya pudiesen resolver sus actividades, si tienen alguna duda relacionado con python, encontré este sitio web donde me soluciona las dudas que tengo.
es https://ellibrodepython.com/ en relación con Python  y https://docs.microsoft.com/es-es/learn/ para gitHub y git, por lo que pueden estudiar la ruta de aprendizaje y les presta por un limite de tiempo un servidor de azure.

Saludos a todos ustedes y si quieren compartir para la comunidad, adelante. espero que sea de utilidad a todos los que se les complica

**Comment by AngelLomas4** [comment]
Muchas gracias, añadidos a Favoritos.
